apps/site/lib/site_web/templates/partial/_expandable_block.html.eex

```diff
@@ -0,0 +1,34 @@
+
+<% container_id = "container-#{@id}" %>
+<% header_id = "header-#{@id}" %>
+<% panel_id = "panel-#{@id}" %>
+
+<div id="<%= container_id %>">
+  <h2 class="c-expandable-block__header">
+    <a class="c-expandable-block__link"
+      href="#<%= panel_id %>"
+      tabIndex="0"
+      id="<%= header_id %>"
+      aria-expanded="false"
```

Comment:
This should be true if `@show_expanded` (and false otherwise)

Reply:
`@show_expanded` is the flag to expand or leave the block collapsed on page load, not a boolean to control the status of the block (perhaps it should be renamed to `@show_expanded_on_load` then?). `aria-expanded="false"` sets all blocks as collapsed initially, then the new jQuery function triggers the click to expand the ones we want. This was the only way I could get the carets to be in sync regardless of whether the block was expanded or collapsed. I don't think this is the best approach so I'm very open to other suggestions!